2/2.2.md

```diff
@@ -73,7 +73,7 @@ Below are a set of steps that one would follow to achieve this
 - Run spec to assert that it fails.
     - Recompile without precompiling the assets, run this if you've changed any Java files
     ```bash
-    ~/projects/go $ SKIP_WAR=Y ./bn clean cruise:prepare
+    ~/projects/go $ ./gradlew clean cruise:prepare -PfastBuild
```

Comment:
This is not needed, running rspecs will automatically compile all any and all java code now :)

The rspec instructions below have changed too —

```
$ ./gradlew rspec # run all specs
$ ./gradlew rspec -Popts='arguments' # to pass args to rspecs
$ ./gradlew rspec -Popts='--pattern spec/controllers/api_v**/*_spec.rb' # to run api specs
$ ./gradlew rspec -Popts='--pattern spec/controllers' # to run controller specs
$ ./gradlew rspec -Popts='--pattern spec/foo/bar_spec.rb' # to run a single spec
```